nsv13/code/modules/overmap/weapons/projectiles_fx.dm

```diff
@@ -563,12 +563,31 @@ Misc projectile types, effects, think of this as the special FX file.
 	tracer_type = /obj/effect/projectile/tracer/disabler
 	muzzle_type = /obj/effect/projectile/muzzle/disabler
 	impact_type = /obj/effect/projectile/impact/disabler
+	relay_projectile_type = /obj/item/projectile/beam/laser/heavylaser/phaser/relayed
+
+/obj/item/projectile/beam/laser/heavylaser/phaser/relayed
+	projectile_piercing = PASSGLASS|PASSGRILLE|PASSTABLE
+	damage = 40 //yes, the explosion will probably gib them. but if it doesn't, let's give them a chance to live, instead of smiting them with the full wraith of the Enterprise 
```

Comment:
Non-devastation explosions cannot gib humans, so this comment is a slight bit wrong.

Reply:
ah. fixed it